sdk/eventhub/azure-eventhubs/azure/eventhub/error.py

```diff
@@ -128,3 +128,84 @@ class EventDataSendError(EventHubError):
 
     """
     pass
+
+
+def _handle_exception(exception, retry_count, max_retries, closable, log):
+    type_name = type(closable).__name__
+    if isinstance(exception, KeyboardInterrupt):
+        log.info("{} stops due to keyboard interrupt".format(type_name))
```

Comment:
Logging statements shouldn't use `.format()` - instead the should use the string formatting like on line 147:
```
log.error("Event data error (%r)", exception)
```

Reply:
fixed